js/marquee.js

```diff
@@ -0,0 +1,35 @@
+class Marquee {
+  constructor(marqueeDiv) {
+    this.marqueeDiv = marqueeDiv;
+  }
+
+  async load() {
+    this.result = await this.priceApi(
+      "https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/quotes/NASDAQ"
```

Comment:
Instead of rewriting the API URL ("https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/) you should use (declare) baseUrl variable. Remember the DRY principle.